runner/runners/invoke.go

```diff
@@ -65,8 +67,16 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 	var checkout snapshot.Checkout
 	var err error
 	go func() {
-		checkout, err = inv.filer.Checkout(cmd.SnapshotID)
-		checkoutCh <- checkoutAndError{checkout, err}
+		if cmd.SnapshotID == "" {
```

Comment:
Ugh, should we just enforce everything needs a snapshot now? If you don't want to go far, at least TODO; this seems like the wrong place in the stack to deal with this if we do want to allow an empty one....